app/WebParsing/ArtSciParser.hs

```diff
@@ -14,51 +14,52 @@ import WebParsing.ParsecCombinators (text)
 import Network.HTTP.Simple (parseRequest, getResponseBody, httpLBS)
 import qualified Text.HTML.TagSoup as TS
 import Text.HTML.TagSoup (Tag)
-import Text.HTML.TagSoup.Match (tagOpenAttrLit, tagOpenAttrNameLit, tagText)
 import Database.Persist.Sqlite (runSqlite, SqlPersistM, insertMany_)
 import Database.Persist (insertUnique)
 import Database.CourseInsertion (insertCourse)
 import Database.Tables (Courses(..), Department(..), Building(..))
 import WebParsing.ReqParser (parseReqs)
 import Config (databasePath)
 import WebParsing.PostParser (addPostToDatabase)
-
+import Data.CSV
+import Text.ParserCombinators.Parsec (parseFromFile)
+import Text.HTML.TagSoup.Match (tagOpenAttrLit, tagOpenAttrNameLit)
+import System.Directory (getCurrentDirectory)
+import Filesystem.Path.CurrentOS as Path
 
 -- | The URLs of the Faculty of Arts & Science calendar.
 fasCalendarURL :: String
 fasCalendarURL = "https://fas.calendar.utoronto.ca/"
 programsURL :: String
 programsURL = "https://fas.calendar.utoronto.ca/listing-program-subject-areas"
-buildingsURL :: String
-buildingsURL = "http://map.utoronto.ca/utsg/c/buildings"
+
+-- The file name is building.csv and it is in the main courseography folder
+buildingsCSV :: IO Prelude.FilePath
```

Comment:
Let me know if you would like this csv file moved to a different location!

Reply:
Put it under `db/`. You might need to change the `.gitignore` to only ignore the sqlite3 files under that folder.